gcp-out-of-cluster.md

```diff
@@ -43,7 +43,20 @@ In Kubecost, navigate to the settings page and click "update" for the "External
 
 <a name="bq-name"></a>**BigQuery dataset** requires a BigQuery dataset prefix (e.g. billing_data) in addition to the BigQuery table name. A full example is `billing_data.gcp_billing_export_v1_018AIF_74KD1D_534A2`.
 
-These config values can alternatively be provided via a [values file](https://github.com/kubecost/cost-analyzer-helm-chart/blob/c10e9475b51612d36da8f04618174a98cc62f8fd/cost-analyzer/values.yaml#L572-L574).
+### Configuring using values.yaml (Recommended)
+
+We recommending providing the GCP details in the [values file](https://github.com/kubecost/cost-analyzer-helm-chart/blob/c10e9475b51612d36da8f04618174a98cc62f8fd/cost-analyzer/values.yaml#L572-L574)  to ensure they are retained during a upgrade or redeploy.
+
+* Set `.Values.kubecostProductConfigs.projectID = <GCP Project ID that contains the BigQuery Export>`
+* Set `.Values.kubecostProductConfigs.gcpSecretName = <Name of the Kubernetes secret that contains the compute-viewer-kubecost-key.json file>`
+* Set `.Values.kubecostProductConfigs.bigQueryBillingDataDataset = <DATASET.TABLE_NAME that contains the billing export>`
+
+Create a secret for the GCP service account key
+> Note: When managing the service account key as a Kubernetes secret, the secret must reference the service account key json file, and that file must be named `compute-viewer-kubecost-key.json`.
+
+``` sh
+kubectl create secret generic gcp-secret -n kubecost --from-file=./compute-viewer-kubecost-key.json
+```
```

Comment:
I think it would be very helpful to put this step above the helm configuration so that it is immediately obvious what `Name of the Kubernetes secret that contains the compute-viewer-kubecost-key.json file` means.

Reply:
Ah. That makes way more sense. Updated. Thanks!